scriptoocloud/pom.xml

```diff
@@ -14,12 +14,21 @@
         <thymeleaf.version>3.0.11.RELEASE</thymeleaf.version>
 
         <admin.password>$2a$04$oQo44vqcDIFRoYKiAXoNheurzkwX9dcNmowvTX/hsWuBMwijqn44i</admin.password>
+         
+        <db.driver>com.mysql.cj.jdbc.Driver</db.driver>
+        <db.database.url>${db-url}</db.database.url>
+        <db.user>${db-user}</db.user>
+        <db.password>${db-password}</db.password>
         
-        <db.driver>com.mysql.jdbc.Driver</db.driver>
-        <db.database.url>jdbc-url-to-db</db.database.url>
-        <db.user>db-user</db.user>
-        <db.password>db-password</db.password>
-
+        <email.user>${email-user}</email.user>
+        <email.password>${email-password}</email.password>
+        <email.ServerHost>${email-ServerHost}</email.ServerHost>
+        <email.ServerPort>${email-ServerPort}</email.ServerPort>
+        <email.emailFrom>${email-emailFrom}</email.emailFrom>
+        <email.instanceUrl>${email-instanceUrl}</email.instanceUrl>
+        <email.appName>${email-appName}</email.appName>
+        <email.tls>${email-tls}</email.tls>
```

Comment:
as discussed yesterday, here you would configure default values that you can override in settings.xml